openmetadata-ui/src/main/resources/ui/src/utils/Alerts/AlertsUtil.tsx

```diff
@@ -13,7 +13,8 @@
 
 import { Typography } from 'antd';
 import { RuleObject } from 'antd/lib/form';
-import i18next from 'i18next';
+import i18next, { t } from 'i18next';
```

Comment:
Can we use either `i18next` or `t` not both?

Reply:
sure